lib/internal/Magento/Framework/Filesystem/Io/File.php

```diff
@@ -564,7 +573,9 @@ public function checkAndCreateFolder($folder, $mode = 0777)
             $this->checkAndCreateFolder(dirname($folder), $mode);
         }
         if (!is_dir($folder) && !$this->mkdir($folder, $mode)) {
-            throw new \Exception("Unable to create directory '{$folder}'. Access forbidden.");
+            throw new LocalizedException(
+                ("Unable to create directory '{$folder}'. Access forbidden.")
```

Comment:
I believe there should be __() or new Phrase...

Reply:
Thank